test/master-playlist-controller.test.js

```diff
@@ -212,6 +212,50 @@ QUnit.test('resets SegmentLoader when seeking in flash for both in and out of bu
 
   });
 
+QUnit.only('selects lowest bitrate rendition when enableLowInitialPlaylist is set',
+  function(assert) {
+    this.player = createPlayer({ html5: { hls: { enableLowInitialPlaylist: true } } });
```

Comment:
since the initial selector is called within the `loadedplaylist` event handler which fires everytime a live playlist is refreshed, I think it would be worth simulating a live refresh and confirming that the initial selector is not called a second time